### PR TITLE
repository-springboot ability to deploy on Heroku

### PR DESCRIPTION
**Pull request by tsurdilo**

### CODE REVIEW

**Comment by tsurdilo** [comment]
@MarianMacik hello again :) mind having a look? Description on how to deploy on heroku is in the readme file updates

File: repository-springboot/pom.xml [hunk 1]
```diff
@@ -172,6 +172,22 @@
...
+        <version>2.0.3</version>
+        <configuration>
+          <appName>${artifactId}</appName>
+          <includeTarget>false</includeTarget>
+          <includes>
+            <include>${project.build.directory}/${project.build.finalName}.jar</include>
+          </includes>
+          <jdkVersion>${java.version}</jdkVersion>
+          <processTypes>
+            <web>java $JAVA_OPTS -jar target/${project.build.finalName}.jar</web>
```
Comment (MarianMacik) [inline repository-springboot/pom.xml]: You have specified `<includeTarget>false</includeTarget>` so is `target` needed in the command? I haven't tried it, but it shouldn't work I guess if target is not included.

File: repository-springboot/README.md [hunk 2]
```diff
@@ -29,4 +29,35 @@ Now you can access the repository at :
...
 
-You can change the default port and context path via src/main/resources/application.properties.
\ No newline at end of file
+You can change the default port and context path via src/main/resources/application.properties.
+
+Running the repository on Heroku
+------------------------------
+Heroku requirements:
+1. Create account and log in
+2. Create app called "repository-springboot". If you want to use a different app name, change the appName value in pom.xml under the heroku-maven-plugin.
```
Comment (MarianMacik) [inline repository-springboot/README.md]: Or a user can provide `-Dheroku.appName` when deploying to Heroku - I think it is better than writing to pom.xml. Wdyt?

File: repository-springboot/pom.xml (hunk 1 above)
Comment (tsurdilo) [reply repository-springboot/pom.xml]: This is the configuration when wanting to deploy only the executable jar which is what we wanted to do here.

**Comment by tsurdilo** [comment]
@MarianMacik updated for -Dheroku.appName . thanks!

File: repository-springboot/pom.xml (hunk 1 above)
Comment (MarianMacik) [reply repository-springboot/pom.xml]: so it will actually upload it to heroku with target/<springboot>.jar directory structure? Then OK.

**Pull request merged**